### PR TITLE
Add localized string for Import settings on Splash Screen

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

Add localized string for Import settings on Splash Screen

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@QilongTang

### CODE REVIEW

File: src/DynamoSandbox/Properties/Resources.en-US.resx [hunk 1]
```diff
@@ -135,6 +135,9 @@
   <data name="SplashScreenFailedImportSettings" xml:space="preserve">
     <value>Failed to Import Settings</value>
   </data>
+  <data name="SplashScreenImportSettings" xml:space="preserve">
+    <value>Import settings</value>
```
Comment (QilongTang) [inline src/DynamoSandbox/Properties/Resources.en-US.resx:139]: Do we need to capitalize it? `Import Settings`?

**Review by QilongTang — APPROVED** [review]

File: src/DynamoSandbox/SplashScreen.xaml.cs [hunk 2]
```diff
@@ -111,6 +111,7 @@ internal async void SetLabels()
             await webView.CoreWebView2.ExecuteScriptAsync("window.setLabels({" +
                $"welcomeToDynamoTitle: '{Properties.Resources.SplashScreenWelcomeToDynamo}'," +
                $"launchTitle: '{Properties.Resources.SplashScreenLaunchTitle}'," +
+               $"importSettingsTitle: 'dd{Properties.Resources.SplashScreenImportSettings}'," +
```
Comment (QilongTang) [inline src/DynamoSandbox/SplashScreen.xaml.cs:114]: typo here `dd`

Comment (zeusongit) [reply src/DynamoSandbox/SplashScreen.xaml.cs:114]: Ahh.. I was testing if the string does update from Dynamo to splash screen. Looks like this was left out, Will submit a fix. Sorry

Comment (zeusongit) [reply src/DynamoSandbox/SplashScreen.xaml.cs:114]: https://github.com/DynamoDS/Dynamo/pull/13466

**Pull request merged**